Source/ApiTemplate/.template.config/template.json

```diff
@@ -694,17 +694,35 @@
   "postActions": [
     {
       "actionId": "3A7C4B45-1F5D-4A30-959A-51B88E82B5D2",
-      "condition": "(!SkipOpenToDo)",
+      "condition": "(!SkipOpenToDo) && (OS == \"Windows_NT\")",
```

Comment:
So essentially if the customer is on Windows and using PowerShell, is there a degraded experience? Does Power Shell automatically "do something" when you use the templates using Windows, that would be lacking on non-Windows side? Not a complaint - just curious - we *try* to retain fidelity across platforms. Not always possible - your effort to do so is applauded especially on such short and informal notice - but if there's a way I'd love to see it achieved.

Reply:
The template comes with a `TODO.html` file which contains a checklist of improvements you can/should make. We try to open this file in your default browser when you create a new project from the template. You can totally, stop this from happening and skip this step by passing the `--no-open-todo` argument to `dotnet new`.

![image](https://user-images.githubusercontent.com/2270364/109038848-12711300-76c4-11eb-961d-f0fd4996b4b4.png)

We use a mechanism given to us by the `dotnet new` templating engine to execute arbitrary shell commands to achieve this. This PR, continues to use PowerShell to open `TODO.html` on Windows and adds the ability to use bash on non-Windows operating systems.